flink-python/src/main/java/org/apache/flink/table/runtime/operators/python/scalar/arrow/ArrowPythonScalarFunctionOperator.java

```diff
@@ -38,7 +38,7 @@
 
 	private static final long serialVersionUID = 1L;
 
-	private static final String SCHEMA_ARROW_CODER_URN = "flink:coder:schema:scalar_function:arrow:v1";
```

Comment:
Why change this?

Reply:
The arrow coder will also be used in Pandas UDAF. It will be strange if scalar function is in coder urn. In fact, the content of the function in the coder urn is not reasonable, but for compatibility, I agree not to modify the urn of the original arrow coder, and then I will add a new urn, and also map it to the arrow coder. What do you think of this?